### PR TITLE
(fix) 'Install Metamask' button

### LINKED ISSUE #478: 'Install Metamask' button is enabled when 'Terms' box is unchecked
1. Disable metamask in browser
2. call the connection modal 

**AR**: 'Install Metamask' button is enabled when 'Terms' box is unchecked

**ER**: 'Install Metamask' button should be disabled when 'Terms' box is unchecked

### PR DESCRIPTION
**Pull request by gabitoesmiapodo**
Closes #478

Link removed.

### CODE REVIEW

File: src/components/modals/WalletModal/index.tsx [hunk 1]
```diff
@@ -195,18 +194,7 @@ const WalletModal: React.FC = () => {
 
       if (option.connector === injected) {
         if (!(window.web3 || window.ethereum)) {
-          if (option.name === 'MetaMask') {
-            return (
-              <Option
-                icon={MetamaskIcon}
-                key={key}
-                onClick={() => window.open('https://metamask.io/')}
```
Comment (josojo) [inline src/components/modals/WalletModal/index.tsx:203]: I think providing the link is to metamask is quite useful. I would not want to remove it.

I would prefer to disable the button, until the user agrees to terms and then still continue to show this link.

Comment (gabitoesmiapodo) [reply src/components/modals/WalletModal/index.tsx:203]: That could end up being kind of confusing: The user has to accept the T&C to enable a button which is a link to install Metamask (accepting the T&C to install an external plugin is weird IMO), which - after installing it- will be the button to connect to Metamask... that's convoluted.

A better (or just simpler) option could be a line of text at the bottom, something like "Don't have a wallet? Try Metamask!" could do it.

Comment (josojo) [reply src/components/modals/WalletModal/index.tsx:203]: >A better (or just simpler) option could be a line of text at the bottom, something like "Don't have a wallet? Try Metamask!" could do it.

you are right. Probably that is simpler.

**Review by hpmaxi — APPROVED** [review]

File: src/components/modals/WalletModal/index.tsx (hunk 1 above)
Comment (gabitoesmiapodo) [reply src/components/modals/WalletModal/index.tsx:203]: Should look like this now.

2021-04-27 15_15_14-Gnosis Auction

**Review by josojo — APPROVED** [review]
Perfect yes. This is a good solution

**Pull request merged**